libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyManagerTests.java

```diff
@@ -38,7 +38,7 @@
 public class PolicyManagerTests extends ESTestCase {
     /**
      * A module you can use for test cases that don't actually care about the
-     * entitlements module.
+     * entitlement module.
      */
     private static Module NO_ENTITLEMENTS_MODULE;
```

Comment:
... like this one.

Reply:
I think it's good to rename them when we see them (or maybe try and address all of them on a simple "renaming" PR?)

Reply:
I'm good either way.